apps/daimo-mobile/src/view/sheet/DepositAddressBottomSheet.tsx

```diff
@@ -0,0 +1,122 @@
+import { assert } from "@daimo/common";
+import { daimoChainFromId } from "@daimo/contract";
+import Octicons from "@expo/vector-icons/Octicons";
+import * as Clipboard from "expo-clipboard";
+import React, { useCallback, useContext, useState } from "react";
+import { StyleSheet, TouchableHighlight, View } from "react-native";
+
+import { DispatcherContext } from "../../action/dispatch";
+import { env } from "../../logic/env";
+import { Account } from "../../model/account";
+import { CheckLabel } from "../shared/Check";
+import { ScreenHeader } from "../shared/ScreenHeader";
+import Spacer from "../shared/Spacer";
+import { color, ss, touchHighlightUnderlay } from "../shared/style";
+import { DaimoText, TextBold, TextLight, TextPara } from "../shared/text";
+import { useWithAccount } from "../shared/withAccount";
+
+export function DepositAddressBottomSheet() {
```

Comment:
recommend a 1-sentence comment above components like this saying what it does

eg > `// Explains how to deposit money directly to your Daimo address`

Reply:
done